packages/es-components/src/components/containers/popover/Popover.js

```diff
@@ -1,28 +1,25 @@
-/* eslint-disable no-confusing-arrow */
-
 import React, { useState, useRef, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
 import Icon from '../../base/icons/Icon';
 import Button from '../../controls/buttons/Button';
+import LinkButton from '../../controls/buttons/LinkButton';
+import OutlineButton from '../../controls/buttons/OutlineButton';
```

Comment:
I'll get back to you on whether it makes sense for popover once I've finished migrating Popup over. For the most part I favor what @jrios did with the notication/calltoaction changes for how to approach button. I liken components built with render props much like the context api; date/state is controlled independently and consumers react to those changes. I don't think this works very well for styling.

Reply:
What do you think about passing a `trigger` prop? You could export a `PopoverTrigger` component that is by default a `LinkButton` with the appropriate styles attached.

Reply:
I am falling more on the side of let the consumers of the component make the decision of what to use rather than trying to guess.

Give as many options as we can.

Reply:
Okay, I've got Popover working with a render prop now. It seems slightly more tricky to use downstream, but it does simplify Popover quite a bit and gives a lot more flexibility. The doc samples should help too.